packages/edit-navigation/src/components/name-editor/index.js

```diff
@@ -34,7 +34,7 @@ export function NameEditor() {
 				label={ __( 'Name' ) }
 				onBlur={ () => setIsMenuNameEditFocused( false ) }
 				className="edit-navigation-name-editor__text-control"
-				value={ editedMenuName }
+				value={ editedMenuName || '' }
```

Comment:
What is this change about?

Reply:
nice catch, it is not needed anymore. I have changed it back.